### ISSUE TITLE
[FORMATTING] PLSQL incorrect format of "for update" syntax 

### ISSUE THREAD

**Issue by AndyGeng123** [sql-formatter-org/sql-formatter#340]
Oracle sql text:
```
 select *  from andy.t_test for update;
```

Expect output:
```sql
 select *  
   from andy.t_test for update;
```

output in demo website:
```sql
   select *
     from andy.t_test for
   update;
```

**Comment by nene** [comment]
Thanks for the bug report.

Unfortunately this is currently pretty hard to fix. Problems like this are the reason why we're looking into doing some major architectural changes in the formatter.

**Comment by nene** [comment]
Found a simpler way of fixing this. Released in 10.2.0.
